### ISSUE TITLE
multi-line ?

### ISSUE THREAD

**Issue by cyrilchapon** [toddbluhm/env-cmd#54]
I tried the third of .env, .json, .js, neither work with a string with newlines inside.

.env
```
TEST=abc\ndef
```

```
TEST="abc
def"
```

.json
```
{ "TEST": "abc\ndef" }
```

.js
```
module.exports = {
  "TEST": `abc
def`
}
```

```
module.exports = {
  "TEST": "abc\ndef"
}
```

None of this works, this is pretty annoying for private keys, especially.

This produces a raw string with `\n` (`\\n` actualy) inside

**Comment by cyrilchapon** [comment]
@toddbluhm 

For now I'm using a combination of `dotenv` + `dotenv-cli` to make this work..

I'm sad because I've been using env-cmd for years now, as it's well maintained (much more than dotenv-cli)

**Comment by mistersdevelopment** [comment]
I was able to work around this issue by base64 encoding my multi-line strings. I know that's not multi-line support but it should help in the meantime.

**Comment by toddbluhm** [comment]
This has been implemented/resoled in the new `9.0.0` release 🎉 Let me know if you still have issues or find other instances where this does not work.
